truffle/src/main/java/org/jruby/truffle/runtime/core/RubyEncoding.java

```diff
@@ -102,4 +102,9 @@ public RubyBasicObject allocate(RubyContext context, RubyClass rubyClass, Node c
 
     }
 
-}
\ No newline at end of file
+    @Override
+    public String toString() {
+        return getName().toString();
```

Comment:
Yeah. If there's a better option, let me know. But I was looking at an array of 100 of them in the debugger and having the `toString` made it much easier to work with.